nevergrad/benchmark/experiments.py

```diff
@@ -11,6 +11,7 @@
 from ..functions import mlda as _mlda
 from ..functions.arcoating import ARCoating
 from ..functions import rl
+from ..games import game
```

Comment:
can you move games into "functions" instead? that's where all the test cases are

Reply:
you can probably also merge functions/rl with it btw

Reply:
Moving to functions/games done.
Merging functions/rl with it: do you mean a single directory for both ?